R/otp-geocode.R

```diff
@@ -0,0 +1,105 @@
+#'  Use OTP Geocoder to find a location
+#'
+#'  Geocoding converts a named place, such as a steet name into a lat/long pair.
+#'
+#' @param otpcon OTP connection object produced by otp_connect()
+#' @param query Character, The query string we want to geocode
+#' @param autocomplete logical Whether we should use the query string to do a prefix match, default FALSE
+#' @param stops Logical, Search for stops, either by name or stop code, default TRUE
+#' @param clusters Logical, Search for clusters by their name, default FALSE
+#' @param corners Logical, Search for street corners using at least one of the street names, default TRUE
+#' @param type Character, How should results be returned can be "SF" or "Coordinates" or "Both", Default "SF"
+#'
+#' @return
+#' Returns a data.frame of SF POINTS or Coordinates of all the locations that match `query`
+#' @examples
+#' locations <- otp_geocode(otpcon, "High Street")
+#' @detials
+#' OTP will return a maximum of 10 results
+#'
+#' @export
+otp_geocode <- function(otpcon = NA,
+                        query = NA,
+                        autocomplete = FALSE,
+                        stops = TRUE,
+                        clusters = FALSE,
+                        corners = TRUE)
+{
```

Comment:
type argument not specified. Function fails.

Reply:
fixed, and switched to checkmate checks